src/OrbitGl/Track.h

```diff
@@ -118,19 +117,16 @@ class Track : public orbit_gl::CaptureViewElement, public std::enable_shared_fro
               const DrawContext& draw_context) override;
   void DoUpdateLayout() override;
 
-  void DrawTriangleFan(Batcher& batcher, const std::vector<Vec2>& points, const Vec2& pos,
-                       const Color& color, float rotation, float z);
   virtual void UpdatePositionOfSubtracks() {}
-  void UpdatePositionOfCollapseToggle();
 
   std::unique_ptr<orbit_accessibility::AccessibleInterface> CreateAccessibleInterface() override;
 
-  bool draw_background_ = true;
   bool pinned_ = false;
   bool headless_ = false;
   uint32_t indentation_level_ = 0;
   Type type_ = Type::kUnknown;
-  std::shared_ptr<TriangleToggle> collapse_toggle_;
+
+  std::shared_ptr<orbit_gl::TrackHeader> header_;
```

Comment:
```suggestion
  std::shared_ptr<TrackHeader> header_;
```

We should be already in the orbit_gl namespace. This does not seem to be the case, which is a bug. Can you fix this locally (i.e. by adjusting just class)?

Reply:
I'd prefer to not do this in the same PR, as it is already quite large.

Reply:
Fine for me.

Reply:
Add a TODO maybe?